Qubit Server/dev/src/org/labrad/qubits/QubitContext.java

```diff
@@ -568,11 +570,17 @@ public void sram_iq_data(@Accepts({"s", "ss"}) Data id,
 	}
 	@SettingOverload
 	public void sram_iq_data(@Accepts({"s", "ss"}) Data id,
-			@Accepts("*c") Data vals,
+			@Accepts({"*c", "(*i{I}, *i{Q})"}) Data vals,
```

Comment:
You're right, but since this is a temporary solution (famous last words) I'd rather leave it as is because this is the return signature of the calibration server function and we're just passing the calibration server results through to this.